bugsnag-android-core/src/main/java/com/bugsnag/android/Logger.kt

```diff
@@ -0,0 +1,12 @@
+package com.bugsnag.android
+
+interface Logger {
+    fun e(msg: String) = Unit
```

Comment:
I noticed that the logger that got removed had fully spelled method names (`debug`, `info` etc.) rather than these single-char abbreviations. Is this definitely the prevailing convention? Would this cause friction for people or is there a way to allow both?

Reply:
This is more consistent with the [default Android logger](https://developer.android.com/reference/android/util/Log), although I think it'd be conventional to name things as either `e()` or `error()`.